http-cors/src/main/scala/org/apache/pekko/http/cors/javadsl/settings/CorsSettings.scala

```diff
@@ -0,0 +1,59 @@
+package org.apache.pekko.http.cors.javadsl.settings
```

Comment:
all source files must have license headers - please use the standard Apache one

Reply:
I swear I saw it somewhere that headers for original/SGA code was a convention but I might be misremembering. I will fix this after I resolve @lomigmegard review comments.

Reply:
the code is granted to us - we can add headers and we need an Apache header

if the original code has had headers, we' d have to keep them or get @lomigmegard 's permission to change them

Reply:
So just to confirm, since the original code has no headers I need to add the standard Apache one?

Reply:
could you use the full header from https://www.apache.org/legal/src-headers.html ?

Reply:
I just updated the PR, I ended up using

```
/*
 * Licensed to the Apache Software Foundation (ASF) under one or more
 * license agreements; and to You under the Apache License, version 2.0:
 *
 *   https://www.apache.org/licenses/LICENSE-2.0
 */
 ```
 
 which is the same as the standard header but without the "derived from Akka" part